inc/customizer/extend-custom-controls/src/sortable/control.js

```diff
@@ -139,14 +147,37 @@ export const sortableControl = wp.customize.astraControl.extend( {
 		control.updateValue();
 	},
 
+	removeSortableItem: function( instance, control ) {
+
+		var parent_wrap = jQuery( instance ).closest( '.ast-sortable-item' ),
+			originalItem = parent_wrap.data( 'index' );
+
+		var mainIndexClonner = control.sortableContainer.find( '.ast-sortable-item[data-value="' + originalItem + '"]' ).find( '.sortable-clonner' );
+
+		if( mainIndexClonner.length ) {
+			mainIndexClonner.show();
+		} else {
+			control.sortableContainer.find( '.ast-sortable-item[data-value="' + originalItem + '"]' ).find( '.dashicons-visibility' ).after( '<i class="dashicons sortable-clonner dashicons-admin-page"></i>' );
+		}
+
+		var sortableItems = control.sortableContainer.find( '.ast-sortable-item[data-index="' + originalItem + '"]' );
+		if( sortableItems.length && 1 < sortableItems.length ) {
+			parent_wrap.remove();
+		}
+
+		control.addListeners( control );
+
+		control.updateValue();
+	},
+
 	addListeners: function( control ) {
-		let cloneTriggers = document.getElementsByClassName( 'clonned-sortable-item' ),
+		let removeTriggers = document.getElementsByClassName( 'remove-sortable-item' ),
 			accordionTriggers = document.getElementsByClassName( 'clonned-sortable-accordion' ),
 			visibilityTriggers = document.getElementsByClassName( 'clonned-sortable-visibility' );
 
-		for( var i=0; i < cloneTriggers.length; i++ ) {
-			cloneTriggers[i].onclick = function() {
-				control.cloneSortableItem( this, control );
+		for( var i=0; i < removeTriggers.length; i++ ) {
```

Comment:
```suggestion
		for( var i=0; i < removeTriggers.length; i++ ) {
```
@imnavanath Please use proper variable names.

Reply:
Done